src/Microsoft.SqlTools.Credentials/Program.cs

```diff
@@ -36,15 +36,12 @@ internal static void Main(string[] args)
                 string logFilePath = commandOptions.LogFilePath;
                 if (string.IsNullOrWhiteSpace(logFilePath))
                 {
-                    logFilePath = "credentials";
-                }
-                if (!string.IsNullOrWhiteSpace(commandOptions.LoggingDirectory))
-                {
-                    logFilePath = Logger.GenerateLogFilePath(Path.Combine(commandOptions.LoggingDirectory, logFilePath));
+                    logFilePath = Logger.GenerateLogFilePath("credentials");
                 }
 
+                Logger.AutoFlush = commandOptions.AutoFlushLog;
```

Comment:
is there any time we wouldn't want auto flush now that it integrates with the output window?

Reply:
This will get exposed as User option with default of false. There is penalty to autoflushing so we will advice user as such. Only where user if ok for some time to play perf penalty and want to see log emitted real time in out output window, will they turn this on.